src/sagemaker_containers/_mpi.py

```diff
@@ -28,6 +28,8 @@
 logger = _logging.get_logger()
 logging.getLogger("paramiko").setLevel(logging.INFO)
 
+AWS_CONTAINER_CREDENTIALS_ENV = "AWS_CONTAINER_CREDENTIALS_RELATIVE_URI"
```

Comment:
Why this particular environment variable? (i.e. Why not any others? Should all environment variables be passed along?)